test/metabase/api/notify_test.clj

```diff
@@ -12,66 +12,63 @@
 (use-fixtures :once (fixtures/initialize :db :web-server))
 
 (deftest unauthenticated-test
-  (mt/with-temporary-setting-values [api-key "testing-api-key"]
```

Comment:
Removed the temporary settings from these tests because `api-key` is already being set as an env var in `project.clj`.